tensorrt_llm/_torch/auto_deploy/custom_ops/linear/swiglu.py

```
@@ -30,12 +30,26 @@
-def _silu_and_mul(x: torch.Tensor) -> torch.Tensor:
-    """SwiGLU activation: split x in half, apply silu to first half, multiply with second half.
 
-    Uses FlashInfer's fused kernel when available, falls back to manual implementation.
-    """
+def _silu_and_mul(x: torch.Tensor) -> torch.Tensor:
+    """SwiGLU activation. Prefer TRT-LLM fused kernel, fall back to flashinfer then manual."""
+    global _trtllm_silu_and_mul, _trtllm_silu_resolved
+    if not _trtllm_silu_resolved:
+        try:
+            _trtllm_silu_and_mul = torch.ops.trtllm.silu_and_mul
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)